packages/app/src/components/pages/VerifyEmail/index.js

```diff
@@ -1,11 +1,24 @@
 import React from 'react'
+import axios from 'axios';
 
-const VerifyEmail = () => {
-    return (
-        <div>
-            <h2>Thank you for verifying your email!</h2>
-        </div>
-    )
-}
+class VerifyEmail extends React.Component {
```

Comment:
Should probably stick to using functional components instead of class components

Reply:
I wanted to use componentDidMount in order to call the api after loading the page, based on your answer I'm assuming there's a way to do that within a functional component?

Reply:
Yeah, check out hooks and useEffect https://reactjs.org/docs/hooks-effect.html

Reply:
cool, thanks